### PR TITLE
FEATURE: Serve RTL versions of admin and plugins CSS bundles for RTL locales

### PR DESCRIPTION
**Pull request by OsamaSayegh**
Prior to this PR, we didn't have RTL versions of our admin and plugins CSS bundles and we always served LTR versions of those bundles even when users used an RTL locale, causing admin and plugins UI elements to never look as good as when an LTR locale was used. Example of UI issues are: missing margins, borders on the wrong side and buttons too close to each other etc.

This PR creates an RTL version for the admin CSS bundle as well as RTL bundles for all the installed plugins and serves those RTL bundles to users/sites who use RTL locales.

Here's the `/admin/customize/themes` page before:

And after:

And this is the chat drawer (from the chat plugin) before:

After:

### CODE REVIEW

File: lib/stylesheet/manager.rb [hunk 1]
```diff
@@ -43,12 +43,21 @@ def self.color_scheme_cache_key(color_scheme, theme_id = nil)
   end
 
   def self.precompile_css
-    targets = %i[desktop mobile desktop_rtl mobile_rtl admin wizard]
+    targets = %i[desktop mobile admin wizard]
+    targets += %i[desktop_rtl mobile_rtl admin_rtl wizard_rtl]
```
Comment (tgxworld) [inline lib/stylesheet/manager.rb:47]:
```suggestion
    targets = %i[
      desktop 
      mobile 
      admin 
      wizard 
      desktop_rtl 
      mobile_rtl 
      admin_rtl 
      wizard_rtl
    ]
```

I don't think appending the RTL items in the array is necessary when we can just declare the array with the RTL items when assigning the variable.

File: app/assets/stylesheets/admin_rtl.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import "admin";
```
Comment (tgxworld) [inline app/assets/stylesheets/admin_rtl.scss:1]: Sorry if I'm missing something but isn't this file just similar to `admin.scss`?

Comment (OsamaSayegh) [reply app/assets/stylesheets/admin_rtl.scss:1]: Yes. `admin_rtl.scss` imports `admin.scss` which imports `common/admin/admin_base.scss` which in turn has a ton of imports and rules that are specific to admin pages.

Our stylesheet compiler expects an identically-named file under `app/assets/stylesheets` for each core bundle, see:

https://github.com/discourse/discourse/blob/d10a050da22d9fc315b8b7ea5e572e1262649c8e/lib/stylesheet/compiler.rb#L25-L39

If we wanted, I think we could special-case bundles that end with `_rtl`, but I think keeping things clear/explicit is a good things plus this allows us to add RTL-specific CSS rules if we need to do that in the future.

Comment (tgxworld) [reply app/assets/stylesheets/admin_rtl.scss:1]: How does adding a `_rtl` postfix at the end adds the RTL-specific CSS rules? :thinking: Do we have a check somewhere which looks for the postfix?

Comment (OsamaSayegh) [reply app/assets/stylesheets/admin_rtl.scss:1]: If we add a rule or an import inside that `admin_rtl.scss` file, then that rule or import will be present in the RTL bundle and not the LTR bundle.

Comment (tgxworld) [reply app/assets/stylesheets/admin_rtl.scss:1]: But there is nothing in the `admin_rtl.scss` file except for `@import "admin"` which is the same as `admin.scss`. Is this PR to prepare for changes that we are going to introduce in the future? I'm also curious how `.pull-left{float:right}` is injected into the RTL scss files.

Comment (OsamaSayegh) [reply app/assets/stylesheets/admin_rtl.scss:1]: Yes, there's no RTL-specific stuff in `admin_rtl.scss` at the moment, but keeping this entry point file for the RTL bundle allows us to do that if we need to. For example, most of the rules in this file:

https://github.com/discourse/discourse/blob/d10a050da22d9fc315b8b7ea5e572e1262649c8e/app/assets/stylesheets/common/base/rtl.scss#L1-L83

are rules that only need to exist in the RTL bundle and not the LTR bundle, but they currently exist in both. With some refactoring, I can fix that so those rules only exist in the RTL bundle, but we need to have a separate file for the RTL bundle that's not the same as the LTR bundle's entry file. We do the same for the mobile and desktop bundle, i.e. we have `mobile_rtl.scss` and `desktop_rtl.scss`.

Comment (tgxworld) [reply app/assets/stylesheets/admin_rtl.scss:1]: Got it :+1: So this PR will make it easier for us to move RTL into the RTL specific files in the future.

Comment (OsamaSayegh) [reply app/assets/stylesheets/admin_rtl.scss:1]: Yes, this PR creates RTL versions for the admin bundle (and plugins), and also allows us to have RTL-specific rules in the RTL version without having them in the LTR versions as well.

**Review by tgxworld — APPROVED** [review]
Changes look good to me :+1:

**Pull request merged**